app/src/main/assets/services.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
I don't think this file belongs in this app. We might use this in tests for loading additional apps though.